priv/cabbage/apps/itest/lib/reorg.ex

```diff
@@ -0,0 +1,163 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule Itest.Reorg do
+  @moduledoc """
+    Chain reorg triggering logic.
+  """
+  use GenServer
+
+  require Logger
+
+  @node1 "geth-1"
+  @node2 "geth-2"
+  @pause_seconds 100
+
+  def finish_reorg() do
+    if Application.get_env(:cabbage, :reorg) do
+      unpause_container!(@node1)
+      unpause_container!(@node2)
+
+      Process.sleep(@pause_seconds * 1000)
+    end
+  end
+
+  def start_reorg() do
+    if Application.get_env(:cabbage, :reorg) do
+      GenServer.cast(__MODULE__, :reorg_step1)
+    end
+  end
```

Comment:
do we need these two?

Reply:
you're right I removed them. Also, I removed genserver since it's not used anymore